easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'protobuf-python'
+version = '3.10.0'
```

Comment:
I wonder how strictly we should stick to this version, also taking into account future TensorFlow version that we'll likely install with `foss/2020a`.

@Flamefire Can we consider a more recent `protobuf-python` here, like latest stable?

Reply:
Current TF master is still using 3.9.2: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/workspace.bzl#L615

I'd say try it ;)

Reply:
It would seem we already have a `protobuf/3.10.0` for `foss/2020a`?

Reply:
Gave it an update anyway, it seems to work as expected!